cmd/ci-secret-bootstrap/main.go

```diff
@@ -833,7 +833,7 @@ func (o *options) validateBWItems(client secrets.ReadOnlyClient) error {
 					continue
 				}
 				if !hasItem {
-					if o.generatorConfig.IsItemGenerated(item.BWItem) {
+					if o.generatorConfig.IsItemGenerated(strings.TrimPrefix(item.BWItem, o.config.VaultDPTPPRefix+"/")) {
```

Comment:
I'd consider giving the `strings.TrimPrefix(item.BWItem, o.config.VaultDPTPPRefix+"/")` a name and put it into a little helper so it does not need to be inline everywhere, for readability. But it's not a big deal.

Reply:
Did that, mostly to have a location for an explanatory comment as this is pretty confusing. Also fixed the tests, PTAL